MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/location/CustomizeUserLocationActivity.java

```diff
@@ -38,7 +38,6 @@ protected void onCreate(Bundle savedInstanceState) {
 
     // Get the location engine object for later use.
     LocationEngine locationEngine = LocationSource.getLocationEngine(this);
-    locationEngine.activate();
     locationEngine.requestLocationUpdates();
```

Comment:
Oh, actually I'll try it now that https://github.com/mapbox/mapbox-android-demo/pull/321#pullrequestreview-36279345 has landed